item-requests.md

```diff
@@ -0,0 +1,43 @@
+# Item Request Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+This supports the Request a Copy feature.
+
+## Main Endpoint
+**/api/copy_request/copy_requests**
+
+Provide access to requests. It returns the list of existing requests.
+
+Example: to be provided
+
+## Single Request
+**/api/copy_request/copy_requests/<:id>**
```

Comment:
I don't like the _ in our URL and I will prefer to use a category name that could be eventually reused.
What about oatools (open access tools), irtools or simply tools?